UnoCheck/Solutions/GTK3NotInstalledOnWindowsSolution.cs

```diff
@@ -0,0 +1,78 @@
+﻿using DotNetCheck.Models;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class GTK3NotInstalledOnWindowsSolution : Solution
+	{
+		public override async Task Implement(SharedState sharedState, CancellationToken cancellationToken)
+		{
+			StringBuilder stringBuilder = new();
+
+			var chocoResult = ShellProcessRunner.Run("powershell.exe", "Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))");
```

Comment:
We can suggest to install a specific version, or install one particular binary "https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases". There's no need to install everything else.

Reply:
Done!